config/sources/mvebu64.conf

```diff
@@ -13,11 +10,27 @@ UBOOT_USE_GCC='> 7.2'
 BOOTENV_FILE='clearfog-default.txt'
 
 ATFSOURCE='https://github.com/MarvellEmbeddedProcessors/atf-marvell'
-ATFDIR='arm-trusted-firmware-espressobin'
+ATFDIR='arm-trusted-firmware-mvebu64'
```

Comment:
@ebin-dev are we safe to use the generic mvebu64 trusted firmware for espressobin?

Reply:
After this change it'll still use atf-marvell one.
ATFDIR is only directory name, where to fetch ATFSOURCE.

I am going to revert it.